addons/pinniped/post-deploy/pkg/configure/configure_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"encoding/base64"
```

Comment:
should we add a test for what happens when `--jwtauthenticator-audience` is passed on the management cluster?

Reply:
Added mgmt cluster 👍